app/controllers/declaration/DocumentsProducedController.scala

```diff
@@ -117,11 +116,7 @@ class DocumentsProducedController @Inject()(
 
   private def updateModelInCache(itemId: String, document: DocumentsProduced, updatedDocs: DocumentsProducedData)(
```

Comment:
Another method here, the same comment like before

Reply:
Disagree on this one... would look horrible to inline this

Reply:
I mean only this line: `updateCache(itemId, journeySessionId, updatedDocs)` beacuase actual function is just wrapper for this method. Argument document is not used in the method.